src/kind_lab_to_nwb/rat_behavioural_phenotyping_2025/prey_capture/nwbconverter.py

```diff
@@ -1,15 +1,46 @@
 """Primary NWBConverter class for this dataset."""
+
+from datetime import timedelta
 from typing import Optional
+from warnings import warn
 
+import numpy as np
 from pynwb import NWBFile
 from pynwb.device import Device
 
+from kind_lab_to_nwb.rat_behavioural_phenotyping_2025.utils import (
+    parse_datetime_from_filename,
+)
 from neuroconv import ConverterPipe
 
 
 class PreyCaptureNWBConverter(ConverterPipe):
     """ConverterPipe for the prey capture dataset."""
 
+    def temporally_align_data_interfaces(self, metadata, conversion_options: Optional[dict] = None):
+        video_interfaces = [
+            interface_name for interface_name in self.data_interface_objects if "Video" in interface_name
+        ]
+        session_start_time = metadata["NWBFile"]["session_start_time"]
+        for interface_name in video_interfaces:
+            video_interface = self.data_interface_objects[interface_name]
+            video_file_path = video_interface.source_data["file_paths"][0]
+            datetime_from_filename = parse_datetime_from_filename(video_file_path.name)
+            if datetime_from_filename.tzinfo is None and session_start_time.tzinfo is not None:
+                datetime_from_filename = datetime_from_filename.replace(tzinfo=session_start_time.tzinfo)
+            aligned_starting_time = (datetime_from_filename - session_start_time).total_seconds()
+            video_timestamps = video_interface.get_timestamps(stub_test=True)
```

Comment:
This shouldn't be hardcoded to True

Reply:
see [81d254d](https://github.com/catalystneuro/kind-lab-to-nwb/pull/52/commits/81d254da574ff1bbd0df60a26c2d36cb1c5dc0a3)